cmake-edit-ui/src/main/java/ru/urururu/cmakeedit/ui/CmakeTextPane.java

```diff
@@ -67,8 +67,8 @@ private void parseAll() {
         try {
             fileNode = Parser.parse(new DocumentParseContext(styledDocument), Parser.ErrorHandling.NodesBefore);
         } catch (ParseException e) {
-            // can't be thrown since we're using Parser.ErrorHandling.NodesBefore
-            return;
+            fileNode = null;
+            e.printStackTrace(); // todo remove!
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1148)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)